### DISCUSSION TITLE
Datatable: Filters are not working in column group context

### DISCUSSION THREAD

**Discussion by bouyass** [primefaces/community#522] [PrimeVue]
**Describe the bug**
When using ColumnGroups in a DataTable for the header, filters inputs don't behave well 

If filterDisplay is set as row, the filters row is not shown, so no way to filter.

As this [ source code ](https://github.com/primefaces/primevue/blob/b58ef712dbcb4984a9bf5a3d214bdb3bdf087fdc/components/datatable/TableHeader.vue#L80) shows, there is no a filter that is handled inside the column group branch condition

**Comment by tugcekucukoglu** [comment]
ColumnGroup does not currently support filtering. We could investigate what we can do.

**Reply by dipbhi** [reply]
I need to filter only one column when using ColumnGroup. Is there any work-around using another global level filter?

**Comment by Razikus** [comment]
+1 i need it
